### ISSUE TITLE
Documentation Update: New 'org' endpoint 'createFirst' / new spacecmd option 'org_createfirst'

### ISSUE THREAD

**Issue by wombelix** [uyuni-project/uyuni-docs#1227]
Related to https://github.com/uyuni-project/uyuni/pull/4286

New spacecmd option need to be added to:
https://www.uyuni-project.org/uyuni-docs/en/uyuni/reference/spacecmd/org.html

New xmlrpc api endpoint need to added to:
https://www.uyuni-project.org/uyuni-docs-api/uyuni/api/org.html#apidoc-org

Issue will be updated with a proposal / example as soon related PR is in review.

**Comment by wombelix** [comment]
The Feature was implemented and merged into the master branch.

I tried to figure out how to create a useful PR here in the docs repo but have to admit, most of the tooling is new to me so I would appreciate if someone else can take care about updating the right files.

I'm happy to help with the necessary content though.

Example for https://www.uyuni-project.org/uyuni-docs/en/uyuni/reference/spacecmd/org.html
```
org_createfirst

Create the initial organization and admin user after completing the installation

usage: org_createfirst [options]

options:
  -n ORG_NAME
  -u USERNAME
  -f FIRST_NAME
  -l LAST_NAME
  -e EMAIL
  -p PASSWORD
```

Will https://www.uyuni-project.org/uyuni-docs-api/uyuni/api/org.html#apidoc-org be generated based on the javadoc comment or do I have to provide an Example?

See: https://github.com/uyuni-project/uyuni/pull/4286/files#diff-f1985bdcf909fc47b52db398cb66580aa3210c14851e15c4bbd00d27731f1e5b

**Comment by wombelix** [comment]
@keichwa I think this one is ready to be moved into `docs backlog` and that `feature not ready` label get removed. I don't think I have the permissions to do that, so maybe you can take a look again? I also made suggestions about the changes, please let me do if I can do anything else. 
I saw that the api-docs are already updated, probably based on the java docs, so it's only about adding the new feature to the spacecmd reference.

**Comment by keichwa** [comment]
> I also made suggestions about the changes, please let me do if I can do anything else. I saw that the api-docs are already updated, probably based on the java docs, so it's only about adding the new feature to the spacecmd reference.

Thanks!  I referenced it in the in our backlog.  Of course, we'd appreciate if you could create a documentation PR if time permits. We have rough hints about PRs in our uyuni docs wiki.  We can provide more help if needed!

**Comment by keichwa** [comment]
> Example for https://www.uyuni-project.org/uyuni-docs/en/uyuni/reference/spacecmd/org.html

Thanks for providing all the input info.  The PR is now pending here (it is just a literal copy, but feel free to check it): https://github.com/uyuni-project/uyuni-docs/pull/1545


> Will https://www.uyuni-project.org/uyuni-docs-api/uyuni/api/org.html#apidoc-org be generated based on the javadoc comment or do I have to provide an Example?

Yes, this one will be generated based on the javadoc comment `:-)`

**Comment by wombelix** [comment]
> > Example for https://www.uyuni-project.org/uyuni-docs/en/uyuni/reference/spacecmd/org.html
> 
> Thanks for providing all the input info. The PR is now pending here (it is just a literal copy, but feel free to check it): #1545
> 

Thank you very much, I wasn't exactly sure what and where to edit, so the least I could do was providing something you could directly use :)

> > Will https://www.uyuni-project.org/uyuni-docs-api/uyuni/api/org.html#apidoc-org be generated based on the javadoc comment or do I have to provide an Example?
> 
> Yes, this one will be generated based on the javadoc comment `:-)`

Perfect, something learned for the next contribution, looking forward and next time I will definitely give it a try to raise the documentation PR as well.

**Comment by keichwa** [comment]
Now is is merged.  To be published with the next release (2022.05 or later).
